src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.IToolForm.cs

```diff
@@ -124,7 +124,8 @@ public override void Restart()
 				}
 				else
 				{
-					RefreshDialog();
+					TastudioStopMovie();

```

Comment:
As this is the same first call as in the other if branch, might as well merge it outside of the if statement.